tests/unit/EditShareStructure.spec.ts

```diff
@@ -149,7 +149,7 @@ describe('Edit Share Structure component', () => {
     const wrapper: Wrapper<EditShareStructure> = createComponent(shareClass, -1, 1, null, [])
     await Vue.nextTick()
 
-    wrapper.find(doneButtonSelector).trigger('click')
+    await wrapper.find(doneButtonSelector).trigger('click')
     await Vue.nextTick()
```

Comment:
Since you're now awaiting the trigger, you may not need to await until the next tick. Ditto below.

Reply:
I tried without nextTick, still didn't work

Reply:
OK, sometimes that happens. No worries.